test/transmuxer.test.js

```diff
@@ -1967,22 +1973,32 @@ function() {
   videoSegmentStream.push({
     data: new Uint8Array([0x09, 0x01]),
     nalUnitType: 'access_unit_delimiter_rbsp',
-    dts: 150,
-    pts: 160
+    dts: lastFrameStartDts,
+    pts: lastFrameStartPts
   });
   videoSegmentStream.flush();
 
   QUnit.equal(segmentTimingInfoArr.length, 1, 'triggered segmentTimingInfo once');
+  QUnit.equal(
+    baseMediaDecodeTime,
+    segmentTimingInfoArr[0].baseMediaDecodeTime,
+    'set baseMediaDecodeTime'
+  );
   QUnit.deepEqual(segmentTimingInfoArr[0], {
     start: {
-      dts: 50,
-      pts: 60
+      dts: baseMediaDecodeTime,
+      pts: baseMediaDecodeTime + startingPts - startingDts
```

Comment:
I would just calculate out the value here in the expected result as we don't want the tests to repeat the code